packages/itwin/tree-widget/src/e2e-tests/TreeWidget.test.ts

```diff
@@ -30,36 +41,22 @@ test.describe("should match image snapshot", () => {
     const node = locateNode(treeWidget, "ProcessPhysicalModel");
     await node.getByTestId("tree-node-expansion-toggle").click();
 
-    // wait for node at the bottom to be visible/loaded
-    await locateNode(treeWidget, "Tag-Category").waitFor();
+    await new Promise((f) => setTimeout(f, 3000));
```

Comment:
Why `locateNode` was changed to this timeout?

Reply:
I believe the last node never gets loaded when using the expanded layout as it is too far away and needs to be scrolled to.

Reply:
I think we could pass last node label to `testCases` function instead of setting 3s timeout. Can you go over all timeout you added and replace them with check for something to appear in UI?

Reply:
Passed the node label to `testCases` and reverted to wait for the node to appear.